core-tests/src/test/java/org/mule/el/mvel/datatype/AbstractVarAssignmentDataTypePropagatorTestCase.java

```diff
@@ -50,7 +50,7 @@ protected AbstractVarAssignmentDataTypePropagatorTestCase(EnricherDataTypePropag
         this.dataTypePropagator = dataTypePropagator;
     }
 
-    protected void doAssignmentDataTypePropagationTest(PropertyScope scope, String expression) throws Exception
+    protected void doFlowVarAssignmentDataTypePropagationTest(String expression) throws Exception
```

Comment:
Does not make sense to me to have these methods on an parent class as they are used only in an specific subclass (flowVar vs session)